.github/workflows/build-os-image.yml

```diff
@@ -581,18 +581,163 @@ jobs:
           jq -sSc '.[0] * .[1] * .[2] * .[3]' ${{ github.workspace }}/pcr-* > ${{ github.workspace }}/pcrs-${{ matrix.csp }}.json
           echo "::endgroup::"
 
+      - name: Add static PCRs
```

Comment:
I am wondering if there could be a different place than a workflow file where we store the PCRs. Maybe the image folder? 

A file in that folder could hold all measurements in a json. This step then takes just the selected CSP and redirects that section into the `pcrs-$CSP.json` for upload.

Reply:
What is the motivation to have these in a separate JSON file in the image folder? Are there other consumers of this file? 

In general I think it makes sense to have the information as local as possible if there is only a single consumer (currently the pipeline).